src/MLJGLMInterface.jl

```diff
@@ -45,24 +45,30 @@ const LCR_DESCR = "Linear count regressor with specified "*
 # LinearBinaryClassifier --> Probabilistic w Binary target // logit,cauchit,..
 # MulticlassClassifier   --> Probabilistic w Multiclass target
 
+const VALID_KEYS = [:deviance, :dof_residual, :stderror, :vcov, :coef_table]
```

Comment:
So the idea of this PR is that performance can be improved by letting the user specify what statistics have to be calculated?